tools/diffscripts/diffapi.ps1

```diff
@@ -0,0 +1,370 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+<#
+.SYNOPSIS
+Microsoft Azure IoT SDK Release comparison script
+
+.DESCRIPTION
+This script is used to compare the current candidate for release to the previous version that is in the iot-sdk-internals repository
+
+Parameters:
+    -AsmToolExecutable: The path to the AsmDiff tool found in the dotnet arcade (ex: c:\tools\asmdifftool)
+    -SDKInternalsPath: The path of the iot-sdk-internals repository (ex: c:\repo\iot-sdks-internals)
+	-IsPreview indicates you will compare the output to the last preview version
+
+Prerequisites:
+    AsmDiff - Used to create the markdown files we use
+    dotnet tool install Microsoft.Dotnet.AsmDiff -g --add-source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json --version 6.0.0-beta.21161.15
+
+    SDK Internals repository - Place we store our markdown files
+    git clone https://github.com/Azure/iot-sdks-internals.git
+
+.EXAMPLE
+.\diffapi
+
+Executes the commands assuming everything is cloned relative to the SDK repository and the AsmDiff tool is compiled using the arcade instructions.
+.EXAMPLE
+.\diffapi -AsmToolExecutable c:\tools\asmdifftool
+
+Specifies the location of the AsmDiff tool executable to run the diff commands.
+.EXAMPLE
+.\diffapi -SDKInternalsPath c:\repo\iot-sdks-internals
+
+Specifies the location of the iot-sdk-internals repository if not cloned relative to the azure-iot-sdk-csharp repository.
+.EXAMPLE
+.\diffapi -IsPreview
+
+Executes the commands assuming you have cloned the iot-sdk-internals repository to a directory relative to the azure-iot-sdk-csharp repository, and if you've installed the asmdiff tool.
+.LINK
+https://github.com/azure/azure-iot-sdk-csharp
+#>
+
+Param(
+    [ValidateScript( {
+            if (-Not ($_ | Test-Path -PathType Leaf)) 
+            {
+                throw "File $_ does not exist."
+            } 
+            elseif ((Get-Command $_).Extension.ToLower() -ne '.exe') 
+            {
+                throw "File $_ is not an executable."
+            }
+            return $true
+        })]
+    # The executable path to the compiled AsmDiff tool found in the dotnet arcade (ex: c:\tools\asmdifftool\dotnet-asmdiff.exe)
+    [System.IO.FileInfo] $AsmToolExecutable = $null,
+    [ValidateScript( {
+            if (-Not ($_ | Test-Path -PathType Container)) 
+            {
+                throw "Folder $_ does not exist."
+            }
+            return $true
+        })]
+    # The path of the iot-sdk-internals repository (ex: c:\repo\iot-sdks-internals)
+    [System.IO.FileInfo] $SDKInternalsPath = $null,
+    # Indicates you will compare the output to the last preview version instead of master
+    [switch] $IsPreview,
+    # Enable debug logging
+    [switch] $Debug
```

Comment:
`$IsDebug`

Reply:
Removed and changed it to react to -Verbose instead.